src/dmd/statementsem.d

```diff
@@ -2234,146 +2234,22 @@ else
-                        return setError();
-
-                    if (global.params.verbose)
-                    {
-                        message("library   %.*s", cast(int)se.len, se.string);
-                    }
-                }
-            }
-        }
-        else if (ps.ident == Id.startaddress)
-        {
-            if (!ps.args || ps.args.dim != 1)
-                ps.error("function name expected for start address");
-            else
-            {
-                Expression e = (*ps.args)[0];
-                sc = sc.startCTFE();
-                e = e.expressionSemantic(sc);
-                e = resolveProperties(sc, e);
-                sc = sc.endCTFE();
-
-                e = e.ctfeInterpret();
-                (*ps.args)[0] = e;
-                Dsymbol sa = getDsymbol(e);
-                if (!sa || !sa.isFuncDeclaration())
-                {
-                    ps.error("function name expected for start address, not `%s`", e.toChars());
-                    return setError();
-                }
-                if (ps._body)
-                {
-                    ps._body = ps._body.statementSemantic(sc);
-                    if (ps._body.isErrorStatement())
-                    {
-                        result = ps._body;
-                        return;
-                    }
-                }
-                result = ps;
-                return;
-            }
-        }
-        else if (ps.ident == Id.Pinline)
-        {
-            PINLINE inlining = PINLINE.default_;
-            if (!ps.args || ps.args.dim == 0)
-                inlining = PINLINE.default_;
-            else if (!ps.args || ps.args.dim != 1)
-            {
-                ps.error("boolean expression expected for `pragma(inline)`");
-                return setError();
-            }
-            else
-            {
-                Expression e = (*ps.args)[0];
-                if (e.op != TOK.int64 || !e.type.equals(Type.tbool))
-                {
-                    ps.error("pragma(inline, true or false) expected, not `%s`", e.toChars());
-                    return setError();
-                }
-
-                if (e.isBool(true))
-                    inlining = PINLINE.always;
-                else if (e.isBool(false))
-                    inlining = PINLINE.never;
-
-                    FuncDeclaration fd = sc.func;
-                if (!fd)
-                {
-                    ps.error("`pragma(inline)` is not inside a function");
-                    return setError();
-                }
-                fd.inlining = inlining;
-            }
+            result = ps._body = ps._body.statementSemantic(sc);
         }
         else if (!global.params.ignoreUnsupportedPragmas)
         {
-            ps.error("unrecognized `pragma(%s)`", ps.ident.toChars());
-            return setError();
-        }
-
-        if (ps._body)
-        {
-            ps._body = ps._body.statementSemantic(sc);
+            result = null;
```

Comment:
Are you sure this is correct? Shouldn't this condition block be removed?

Reply:
That was supposed to be unconditional. I think I messed up a bit trying to squash a conflicting merge (#8552).